core/src/main/java/cz/xtf/core/waiting/SupplierWaiter.java

```diff
@@ -20,15 +22,15 @@ public SupplierWaiter(Supplier<X> supplier, Function<X, Boolean> successConditio
 	}
 
 	public SupplierWaiter(Supplier<X> supplier, Function<X, Boolean> successCondition, Function<X, Boolean> failureCondition) {
-		this(supplier, successCondition, failureCondition, TimeUnit.MILLISECONDS, DEFAULT_TIMEOUT);
```

Comment:
Either set `DEFAULT_TIMEOUT` to `WaitingConfig.timetout()` or remove `DEFAULT_TIMEOUT` from `Waiter` class.